src/muya/lib/parser/marked/renderer.js

```diff
@@ -170,20 +161,11 @@ Renderer.prototype.del = function (text) {
 }
 
 Renderer.prototype.link = function (href, title, text) {
-  if (this.options.sanitize) {
-    let prot
-    try {
-      prot = decodeURIComponent(unescape(href))
-        .replace(/[^\w:]/g, '')
-        .toLowerCase()
-    } catch (e) {
-      return ''
-    }
-    if (prot.indexOf('javascript:') === 0 || prot.indexOf('vbscript:') === 0) {
-      return ''
-    }
+  href = cleanUrl(this.options.sanitize, this.options.baseUrl, href)
+  if (href === null) {
+    return text
   }
-  let out = '<a href="' + href + '"'
+  var out = '<a href="' + escape(href) + '"'
```

Comment:
If we want to be consistent with the code specification of markedjs, that is, use var to declare variables, then we need to ignore the marked folder in `eslintignore file`

Reply:
I overlooked at, I'll replace all `var`'s.